pylint/lint/pylinter.py

```diff
@@ -938,16 +939,18 @@ def initialize(self):
             if not msg.may_be_emitted():
                 self._msgs_state[msg.msgid] = False
 
-    def check(self, files_or_modules):
+    def check(self, files_or_modules: Union[Sequence[str], str]) -> None:
         """main checking entry: check a list of files or modules from their name.
 
-        files_or_modules is either a string or list of strings presenting modules to check.
+        files_or_modules is a sequence of strings presenting modules to check.
```

Comment:
This should be reverted

Reply:
Weird how Github doesn't allow a suggestion here..

Reply:
I've highlighted both an old an a new line. That's probably why.